app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java

```diff
@@ -65,9 +68,22 @@ protected void onCreate(Bundle savedInstanceState) {
         sessionRecyclerView.setLayoutManager(new LinearLayoutManager(this));
         sessionRecyclerView.setAdapter(sessionsListAdapter);
         sessionRecyclerView.setItemAnimator(new DefaultItemAnimator());
+
+        shareButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                String speakerInfo = ("Speaker is: ".concat(speaker).concat("\n").concat("Speaker's email is: ").concat(selectedSpeaker.getEmail()));
```

Comment:
Not Localization friendly plus has characters that can be hijacked to cause injection problems.